ci/docker-compose-tests.yaml

```diff
@@ -25,9 +25,8 @@ services:
         - no_proxy
     volumes:
```

Comment:
Question: Will this work if the host machine doesn't have the build available? The TCF directory is replaced with the host machine files. I have similar questions for the other files in this commit.

Reply:
Since TCF build is part of docker image with these changes, test case execution will succeed because all build artifacts are already part of docker image.

Reply:
But i have removed volume mapping from the compose file, hence libraries built as part of docker image will be used.

Reply:
Right! After the change it won't map.